src/pages/workspace/WorkspaceNewRoomPage.tsx

```diff
@@ -222,7 +222,7 @@ function WorkspaceNewRoomPage() {
                 success
                 large
                 text={translate('footer.learnMore')}
-                onPress={() => Navigation.navigate(ROUTES.SETTINGS_WORKSPACES)}
+                onPress={() => Navigation.navigate(ROUTES.SETTINGS_WORKSPACES.route)}
```

Comment:
We are using navigate without backTo here causing https://github.com/Expensify/App/issues/59686